Bump php minimum requirement to 8.2

doxigo opened the pull request:
It's a good consideration to start off with php 8.2 with Drupal 10.2 considerations

doxigo committed:
Bump php minimum requirement to 8.2

It's a good consideration to start off with php 8.2 with Drupal 10.2 considerations

jfauske commented:
@doxigo This change requires updates to https://github.com/ramsalt/drupal-project/blob/10.x/.github/workflows/ci.yml too

doxigo committed:
Update ci.yml to use php82

Following up on https://github.com/ramsalt/drupal-project/pull/30, bumping the php version accordingly here as well

doxigo commented:
@jfauske created a mirroring PR there as well https://github.com/ramsalt/drupal-project/pull/31

doxigo committed:
Update the missing version

jfauske commented:
I'll rebase this on top of the bump-php82 branch so we can get the checks passing.
